controllers/course.js

```diff
@@ -0,0 +1,102 @@
+const course = {
+    name: "tema 1",
+    endpoint: "/tema1",
+    content: [
+        {
+            id: 1,
+            title: "El nombre es el propósito/Hasta Jacob.",
+            description:
+                "1 dentidad ¿quiénes somos? ¿De quiénes venimos? ¡Conoce la historia de esta familia de la cuál tú eres parte!",
+            url: "https://www.youtube.com/embed/MrZEACaZ1Z8",
+        },
+        {
+            id: 2,
+            title: " Hasta los jueces / Olvida tu pasado, mira tu futuro.",
+            description:
+                "2 dentidad ¿quiénes somos? ¿De quiénes venimos? ¡Conoce la historia de esta familia de la cuál tú eres parte!",
+            url: "https://www.youtube.com/embed/Fv8aSrgujyM",
+        },
+        {
+            id: 3,
+            title: "Instrucciones para un Rey | El corazón de David revelado",
```

Comment:
Review all the titles please...

Reply:
These titles are temporary while we have the data that will be rendered in production